eventdata/runners/deleteindex_runner.py

```diff
@@ -66,8 +66,9 @@ def get_suffix(name, separator):
     suffix_separator = params.get("suffix_separator", "-")
 
     if max_indices:
-        response = await es.cat.indices(h="index")
-        indices = response.decode("utf-8").split("\n")
+        indices = await es.cat.indices(h="index")
+        if isinstance(indices, bytes):
+            indices = indices.decode("utf-8").split("\n")
```

Comment:
I think you want to split even if you did not have to decode:

```suggestion
        if isinstance(indices, bytes):
            indices = indices.decode("utf-8")
        indices = indices.split("\n")
```

We could also use splitlines() but that could be less explicit.